plugins/optimization-detective/detect.js

```diff
@@ -539,6 +541,7 @@ export default async function detect( {
 
 	const url = new URL( restApiEndpoint );
 	url.searchParams.set( 'slug', urlMetricSlug );
+	url.searchParams.set( 'eTag', currentETag );
```

Comment:
It's a bit confusing that this is here rather than (also?) added as `urlMetric.eTag` (or `urlMetric.etag`). In the end they will need to be identical of course, so it does make sense. Just an observation. Maybe this should be called `current_etag` to better disambiguate?

Reply:
Renamed it in 371e998.

Reply:
Although, I fail to understand the reason for sending ETag as part of the `urlMetric` object as well.